appeals/web/src/server/appeals/appeal-details/service-user/__tests__/service-user.test.js

```diff
@@ -0,0 +1,212 @@
+import { parseHtml } from '@pins/platform';
+import supertest from 'supertest';
+import { appealData } from '#testing/app/fixtures/referencedata.js';
+import { createTestEnvironment } from '#testing/index.js';
+import nock from 'nock';
+
+const { app, installMockApi, teardown } = createTestEnvironment();
+const request = supertest(app);
+const baseUrl = '/appeals-service/appeal-details';
+
+describe('service-user', () => {
+	beforeEach(installMockApi);
+	afterEach(teardown);
+
+	describe('GET /change/:userType', () => {
+		it('should render the change service user page for an agent', async () => {
+			const appealId = appealData.appealId.toString();
+			const response = await request.get(`${baseUrl}/${appealId}/service-user/change/agent`);
+			const element = parseHtml(response.text);
+
+			expect(element.innerHTML).toMatchSnapshot();
+			expect(element.innerHTML).toContain(`Update agent details`);
```

Comment:
Very minor, there's no need to use backticks if nothing is being templated into the string - nothing wrong with it though